pyccl/nl_pt/__init__.py

```diff
@@ -7,19 +7,22 @@
     translate_IA_norm,
 )
 
-# Power spectra
-from .power import (
+# Old power spectrum calculator
+from .power_deprecated import (
     PTCalculator,
     get_pt_pk2d,
 )
 
+# Eulerian PT
+from .ept import EulerianPTCalculator
```

Comment:
Does `EulerianPTCalculator` work with any `PTTracer`? Or might there be a situation in the future where some `*PTCalculator` only works with certain`PTTracer`?

Reply:
`EulerianPTCalculator` does work with any `PTTracer`, but that may not be the case with future calculators (e.g. I'm fairly certain `velocileptors` doesn't do IAs. What are you worried about?

Reply:
I'm thinking about how to make sure that only valid combinations of `PTCalculator` and `PTTracers` are used in something like firecrown. Right now the assumption is that there is one `PTCalculator` that can be used with all the PT tracers but I wonder if there might be a situation in the future where there are multiple `PTCalculator`s that need to be matched with their corresponding tracers.

Reply:
The way I see it, Tracers are fully general, it's just that some calculators will only be able to compute certain quantities. E.g. the LPT calculator will have to throw an error when asked to compute an IA P(k). But all calculators that can compute e.g. Pk_gg will be able to work with the same `PTNumberCountsTracer`s (i.e. there won't be tracers specific for a type of calculator).